spark-api/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@filecoin-station/spark-api",
+  "private": true,
+  "version": "0.0.0",
+  "license": "MIT",
+  "repository": "filecoin-station/spark-api",
+  "type": "module",
+  "description": "API for SPARK",
+  "scripts": {
+    "start": "node bin/spark.js",
+    "test": "mocha"
```

Comment:
In the main branch, one could run `(cd spark-publish && npm test)` to execute both the linter and the tests for a sub-package.

Is this something you want to preserve in the new monorepo setup?

In my proposal, you always have to run the linter from the monorepo root, because that's the place where `package.json` tells `standard` about Mocha.

Reply:
I would prefer indeed if both would work: linting from the parent and the subpackage. Could we create a standard config that re-exports the parent config?

Reply:
Done in https://github.com/filecoin-station/spark-api/pull/326/commits/daa26500505d69998ac67522d8b53eaa72ab5065

Reply:
> Could we create a standard config that re-exports the parent config?

I could not find any quick way how to do that, Standard seems to accept config only via CLI args or in the closest `package.json` file.

The alternative is to eject from standard to eslint, but that's much more work than I am willing to do now.

We could also remove the env config from package.json and modify every file to start with the following line:

```
/* eslint-env mocha */
```

It would be cleaner because non-test files won't have access to `describe` & `it` globals, but it also feels like too much boilerplate.

Reply:
I ended up duplicating the `"standard":{"env":...}}` section in each `package.json`.